src/shared/sam/debugger/goSamDebug.ts

```diff
@@ -137,8 +144,7 @@ export async function makeGoConfig(config: SamLaunchRequestArgs): Promise<GoDebu
         port: port,
         skipFiles: [],
         debugArgs: isImageLambda || config.noDebug ? undefined : ['-delveAPI=2'],
-        localRoot: localRoot ?? config.codeRoot,
-        remoteRoot: remoteRoot ?? '/var/task',
+        debugAdapter: 'legacy', // Just in case the Go extension decides to make Delve DAP the default
```

Comment:
(future note) should/can we write an integration test that fails if this changes? So that we notice it.